### ISSUE TITLE
Is computation_time only related to the GPU model?

### ISSUE THREAD

**Issue by LLJING-ER** [argonne-lcf/dlio_benchmark#219]
I'm trying to create a config file ***.yaml, but confused about the meaning of **computation_time** , is it related to **batchsize**? If I change the **batchsize**, do I need to  run the actual workload to find out the corresponding **computation_time**?

**Comment by zhenghh04** [comment]
Yes it is related to the batch size. It is the computation time for processing one batch of data. So you'll have to measure the computation time in the actual hardware for that.

**Comment by LLJING-ER** [comment]
It is necessary to maintain consistent data format. How do you measure the **computation_time** of NPZ format data in UNET3D? Can you provide me with a reference code?

**Comment by LLJING-ER** [comment]
If possible, please also provide a copy of the test code in **tfrecord** format for the **cosmoflow** workload
